src/main/kotlin/app/revanced/patches/youtube/layout/castbutton/signatures/CastButtonSignature.kt

```diff
@@ -0,0 +1,29 @@
+package app.revanced.patches.youtube.layout.castbutton.signatures
+
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.signature.implementation.method.MethodSignature
+import app.revanced.patcher.signature.implementation.method.annotation.FuzzyPatternScanMethod
+import app.revanced.patcher.signature.implementation.method.annotation.MatchingMethod
+import app.revanced.patches.youtube.layout.castbutton.annotations.CastButtonCompatibility
+import org.jf.dexlib2.AccessFlags
+import org.jf.dexlib2.Opcode
+
+@Name("cast-button-signature")
+@MatchingMethod(
+    "Landroidx/mediarouter/app/MediaRouteButton", "setVisibility" // first one is prolly causing the issue since not sure what to write there

```

Comment:
Why are you hooking `MediaRouteButton#setVisibility`? What is it you're trying to do with this patch?

Reply:
^

i basically tried to do what vanced did in their code